changelog/unreleased/3137

```diff
@@ -0,0 +1,9 @@
+Bugfix: Sorted collaborators column, deduplicate public entry
+
+The collaborators appears a the column in the "shared with others"
```

Comment:
This sentence sounds a bit strange to me 😕 What about "Collaborators column/columns in the "shared with others" list is/are now sorted..." ?

Reply:
fixed. I had mangled the sentence by hitting the wrong vim shortcut...